views/pulsar/v2/helpers/form.html.twig

```diff
@@ -573,7 +573,6 @@ class   | string | A space separated list of class names
 
 #}
 {% macro end(options = {}) %}
-{% spaceless %}
```

Comment:
Wouldn't've thought removing the top level `spaceless` was required? Didn't removing the whitespace control on L:588 resolve the issue alone?